exercises/practice/gigasecond/.docs/instructions.append.md

```diff
@@ -0,0 +1,6 @@
+# Instructions append
+
+  
+[Official Python documentation on the datetime module](https://docs.python.org/3.9/library/datetime.html#module-datetime)
+[datetime objects](https://docs.python.org/3.9/library/datetime.html#datetime.datetime)
+[timedelta objects](https://docs.python.org/3.9/library/datetime.html#timedelta-objects)
```

Comment:
```suggestion
- [Official Python documentation on the datetime module][datetime]
- [datetime objects][datetime.datetime]
- [timedelta objects][datetime.timedelta]

[datetime]: https://docs.python.org/3.9/library/datetime.html#module-datetime
[datetime.datetime]: https://docs.python.org/3.9/library/datetime.html#datetime.datetime
[datetime.timedelta]: https://docs.python.org/3.9/library/datetime.html#timedelta-objects
```

Reply:
Added bullets here to separate the links, and added the reflinks at the bottom of the doc.  If this doesn't read well, feel free to revert the bullets.  The reflinks are part of our spec, and won't show to the end user.